code/__DEFINES/{yogs_defines}/preferences.dm

```diff
@@ -8,10 +8,15 @@
 
 #define GHOST_CKEY	(1<<11)
 
+#define SOUND_VOX	(1<<22)
```

Comment:
Does this need to be 22?

Reply:
https://github.com/yogstation13/Yogstation/blob/28af47f93de3a371b1cfdb5d634801de684a20f9/code/__DEFINES/preferences.dm#L25